### ISSUE TITLE
Unable to sort all elements in px-table-view-sortable-list

### ISSUE THREAD

**Issue by jyotis3** [PredixDev/px-table-view#8]
In px-table-view-sortable-list, I am unbable to sort all elements.
For example, suppose  wen the page loads there are 5 items in the table , I tab on Item 4 and moves it to the place of Item 3, now its just item 4 that can be dragged and dropped rest other elements cannot be dragged & dropped

**Comment by randyaskin** [comment]
@davidrleonard FYI - https://github.com/PredixDev/px-table-view/tree/sortable-polymer

**Comment by davidrleonard** [comment]
Waiting on a patch to merge into the 3rd party library we use for sorting (RubaXa/Sortable#1075). I'll see if we can monkey patch it until that patch lands — not sure if it's possible, but I'll report back.

**Comment by davidrleonard** [comment]
Decided to roll back to Sortable.js 1.4.X while the patch is being reviewed. Also switched to manually handle adding/removing sorted elements in shady DOM world, where it breaks.

Everything should be fixed as of 138f635 (v2.0.7)
